src/js/value.mbt

```diff
@@ -2,11 +2,45 @@
 pub extern type Value
 
 ///|
+#deprecated("Use `Value::from` or `Value::unsafe_into` instead.")
 pub fn Value::cast_from[T](value : T) -> Value = "%identity"
 
-///|
+///| 
+#deprecated("Use `Value::into` or `Value::unsafe_into` instead.")
 pub fn Value::cast[T](self : Value) -> T = "%identity"
 
+///| Create a `Value` from any type, without checking for null or undefined.
```

Comment:
It seems that you wish to enforce the invariant that a `Value` cannot be null or undefined. If that is the case, you may wish to add caveats in docstrings for both the Value type and these two methods.